docs/api.rst

```diff
@@ -2446,6 +2473,60 @@ of :class:`enum.Enum`.
 
         .. versionadded:: 2.0
 
+    .. attribute:: guild_scheduled_event_create
+
+        A scheduled event was created.
+
+        When this is the action, the type of :attr:`~AuditLogEntry.target` is
+        the :class:`GuildScheduledEvent` or :class:`Object` with the ID of the event
+        which was created.
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.name`
+        - :attr:`~AuditLogDiff.description`
+        - :attr:`~AuditLogDiff.privacy_level`
+        - :attr:`~AuditLogDiff.status`
+        - :attr:`~AuditLogDiff.sku_ids`
+
+        .. versionadded:: 2.3
+
+    .. attribute:: guild_scheduled_event_update
+
+        A scheduled event was updated.
+
+        When this is the action, the type of :attr:`~AuditLogEntry.target` is
+        the :class:`GuildScheduledEvent` or :class:`Object` with the ID of the event
+        which was updated.
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.name`
+        - :attr:`~AuditLogDiff.description`
+        - :attr:`~AuditLogDiff.privacy_level`
+        - :attr:`~AuditLogDiff.status`
+        - :attr:`~AuditLogDiff.sku_ids`
+
+        .. versionadded:: 2.3
+
+    .. attribute:: guild_scheduled_event_delete
+
+        A scheduled event was deleted.
+
+        When this is the action, the type of :attr:`~AuditLogEntry.target` is
+        the :class:`GuildScheduledEvent` or :class:`Object` with the ID of the event
+        which was deleted.
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.name`
+        - :attr:`~AuditLogDiff.description`
+        - :attr:`~AuditLogDiff.privacy_level`
+        - :attr:`~AuditLogDiff.status`
+        - :attr:`~AuditLogDiff.sku_ids`
+
+        .. versionadded:: 2.3
+
```

Comment:
We should document the raw events too.

Reply:
Those should be removed actually, because API devs said that they renamed the events and changed the inner payload structure. I don't know any details so the best we can do is to wait.